test/verify/check-session

```diff
@@ -102,7 +102,7 @@ class TestSession(testlib.MachineCase):
             self.write_file("/etc/pam.d/cockpit", "session required pam_env.so user_readenv=1\n", append=True)
         victim_pid = m.spawn("sleep infinity", "sleep.log")
         self.addCleanup(m.execute, f"kill {victim_pid} || true")
-        self.write_file("/home/admin/.pam_environment", f"SSH_AGENT_PID={victim_pid}\n", owner="admin")
+        self.write_file("/home/admin/.pam_environment", f"SSH_AGENT_PID DEFAULT={victim_pid}\n", owner="admin")
```

Comment:
What does that do? This is very much *not* the [documented syntax](https://linux.die.net/man/8/pam_env) -- for ages this has always been `KEY=value`.

Reply:
Check this: https://linux.die.net/man/5/pam_env.conf

In any case, the thing starts to work on fedora-41 with this change, unless I am confused... :)

Reply:
Yes, but this *isn't* `pam_env.conf. This is /etc/environment or an user env file as described in https://linux.die.net/man/8/pam_env :grin:

Reply:
Sooo, the man pages on Fedora and Debian actually say this:

```
user_envfile=filename
   Indicate an alternative .pam_environment file to override the default.
   The syntax is the same as for /etc/security/pam_env.conf. The filename is 
   relative to the user home directory. This can be useful when different
   services need different environments.
```

Debiian and Ubuntu also accept this syntax. And the crashes on fedora-40 might be caused by using the "wrong" syntax, it check-session passes there now with this change. I pull this out into its own PR.

Reply:
Interesting -- This conceptually so wrong and broken beyond repair that it's really good that this gets phased out..